inst/shiny/DiagnosticsExplorer/tests/test-VisitContextModule.R

```diff
@@ -26,9 +26,36 @@ test_that("Visit context page", {
     checkmate::expect_data_frame(getVisitContexDataEnhanced())
     checkmate::expect_character(getVisitContexDataEnhanced()$databaseId)
     checkmate::expect_character(getVisitContexDataEnhanced()$visitConceptName)
-    checkmate::expect_numeric(getVisitContexDataEnhanced()$Before)
-    checkmate::expect_numeric(getVisitContexDataEnhanced()$During)
-    checkmate::expect_numeric(getVisitContexDataEnhanced()$Simultaneous)
-    checkmate::expect_numeric(getVisitContexDataEnhanced()$After)
+
+    
+    # Initializing vectors with column names
+    before_vec <- c("databaseId", "visitConceptName", "Before")
```

Comment:
Please change to be `camelCase` instead of `snake_case`. We use a [common style guide](https://ohdsi.github.io/Hades/codeStyle.html) and try to be consistent across packages to improve interpretability.